.vib/rails/goss/rails.yaml

```
@@ -11,7 +11,7 @@ command:
   # Major.Minor.Patch-Build, such as 1.2.3-4 while the `rails -v` command returns
   # the version in the format of Major.Minor.Patch.Build, such as 1.2.3.4 so we
   # need to convert the environment variable before comparing the two versions
-    exec: {{ .Vars.version.bin_name }} {{ .Vars.version.flag }} | sed '/^[0-9]\+\.[0-9]\+\.[0-9]\+$/ s/$/-0/' | sed "s/\.\([0-9]\)$/-\1/"
+    exec: {{ .Vars.version.bin_name }} {{ .Vars.version.flag }} | sed '/\s[0-9]\+\.[0-9]\+\.[0-9]\+$/ s/$/\.0/' | sed 's/\.\([0-9]\)$/-\1/'
```

What about using this?

```suggestion
    exec: {{ .Vars.version.bin_name }} {{ .Vars.version.flag }} | sed '/^\s*[0-9]\+\.[0-9]\+\.[0-9]\+$/ s/$/\.0/' | sed 's/\.\([0-9]\)$/-\1/'
```

---

Thanks @jotamartos  but that regexp doesn't match the output of `rails --version`. I'll give you some examples:
```console
$ docker run -it --rm --name rails bitnami/rails:7 bash -c "rails --version | sed '/^\s*[0-9]\+\.[0-9]\+\.[0-9]\+$/ s/$/\.0/' | sed 's/\.\([0-9]\)$/-\1/'"
rails 14:54:35.97 
rails 14:54:35.97 Welcome to the Bitnami rails container
rails 14:54:35.98 Subscribe to project updates by watching https://github.com/bitnami/containers
rails 14:54:35.98 Submit issues and feature requests at https://github.com/bitnami/containers/issues
rails 14:54:35.98 

Rails 7.0-5
```
The output should be `7.0.5-0`

---

We checked that this was working when using 4 digits version but not with 3.